testing/downstream_testing/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM sommersoft/pyenv-pytest:latest
```

Comment:
Ownership of this should likely be transferred. If you'd rather not have to maintain Docker Hub, GitHub's container registry is probably an option. Again, base image is here: https://github.com/sommersoft/docker-pyenv-pytest